src/App.js

```diff
@@ -10,8 +10,7 @@ class App extends Component {
     super(props);
 
     this.state = {
-      tasks: [],
-      tasksToDo: []
+      tasks: []
```

Comment:
2) I've remove the separate list of completed tasks as we can get this info from the state of `completed` on each task